config/service_providers.yml

```diff
@@ -161,6 +161,38 @@ production:
     attribute_bundle:
       - email
 
+# Micro-purchase
+  'urn:gov:gsa:saml:2.0.profiles:sp:sso:micropurchase-staging':
+    acs_url: 'https://micropurchase-staging.18f.gov/auth/saml/callback'
+    assertion_consumer_logout_service_url: 'https://micropurchase-staging.gov/auth/saml/logout'
+    sp_initiated_login_url: 'https://micropurchase-staging.18f.gov/login'
+    block_encryption: 'aes256-cbc'
+    cert: 'sp_micropurchase'
+    agency: 'TTS Acquisition'
+    friendly_name: 'Micro-purchase Staging'
+    logo: 'generic.svg'
+    return_to_sp_url: 'https://micropurchase-staging.gov'
+    attribute_bundle:
+      - email
+      - first_name
+      - last_name
+
+
+  'urn:gov:gsa:SAML:2.0.profiles:sp:sso:micropurchase-production':
+    acs_url: 'https://micropurchase.18f.gov/auth/saml/callback'
+    assertion_consumer_logout_service_url: 'https://micropurchase.gov/auth/saml/logout'
+    sp_initiated_login_url: 'https://micropurchase.18f.gov/login'
+    block_encryption: 'aes256-cbc'
+    cert: 'sp_micropurchase'
```

Comment:
the example SP apps don't make this distinction because they won't be running in true production. basically, their definition of production is different. nothing to update in the example apps